yarn.lock

```diff
@@ -3982,6 +3982,11 @@ jest-worker@^26.5.0:
     merge-stream "^2.0.0"
     supports-color "^7.0.0"
 
+jquery@^3.5.1:
```

Comment:
Yikes - I feel like jQuery is a bit heavy for what we're using it for.

Reply:
i think theres probably enough js pipelined to justify it, trying to put a simple structure in place for our js code